packages/sync/src/class-defaults.php

```diff
@@ -1176,4 +1182,67 @@ public static function is_multi_network() {
 	 * @var int 1 for true.
 	 */
 	public static $default_full_sync_sender_enabled = 1; // Should send full sync items.
+
+	/**
+	 * Default Full Sync config
+	 *
+	 * @var array list of module names.
+	 */
+	public static $default_full_sync_config = array(
+		'constants'          => 1,
+		'functions'          => 1,
+		'options'            => 1,
+		'updates'            => 1,
+		'themes'             => 1,
+		'users'              => 1,
+		'terms'              => 1,
+		'posts'              => 1,
+		'comments'           => 1,
+		'term_relationships' => 1,
+	);
+
+	/**
+	 * Send all Items
+	 *
+	 * @var Array chunk limmits per request request.
+	 */
+	const FULL_SYNC_ALL = array(
```

Comment:
Could we improve the naming here to be more meaningful?

Reply:
help me please

Reply:
How about `ICE_CREAM`

Reply:
For realz, how about:

```suggestion
	const DEFAULT_MODULE_CHUNK_CONFIG = array(
```

Reply:
solved 😊 https://github.com/Automattic/jetpack/pull/13963/commits/69933ced3c8e1e14023c91067fde47e019f86f6b